evmbin/src/display/json.rs

```diff
@@ -125,7 +125,6 @@ impl trace::VMTracer for Informant {
 	fn trace_executed(&mut self, gas_used: U256, stack_push: &[U256], mem: &[u8]) {
```

Comment:
I think this function is not use anymore, also :
    display::json::tests::should_trace_create_correctly
    display::json::tests::should_trace_failure
fails for me (for first case it looks like value of the tests was wrong before).

Reply:
Also running the command :  
```parity-evm --code 5A51 --json --gas fffff```
works fine

Reply:
This is part of the interface for `VMTracer`, which is used in executive and externalities. I don't think we can remove them at this moment!

> fails for me (for first case it looks like value of the tests was wrong before)

That's really weird again! Here's what I got on commit https://github.com/paritytech/parity-ethereum/pull/9873/commits/514d0f1b23d696be569e5d181e9663e851bab379, please double check if I got anything wrong:

```
[nix-shell:~/repos/parity/evmbin]$ cargo test
warning: path override for crate `ring` has altered the original list of
dependencies; the dependency on `libc` was either added or
modified to not match the previously resolved version

This is currently allowed but is known to produce buggy behavior with spurious
recompiles and changes to the crate graph. Path overrides unfortunately were
never intended to support this feature, so for now this message is just a
warning. In the future, however, this message will become a hard error.

To change the dependency graph via an override it's recommended to use the
`[replace]` feature of Cargo instead of the path override feature. This is
documented online at the url below for more information.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#overriding-dependencies

    Updating crates.io index
   Compiling evmbin v0.1.0 (/home/sorpaas/repos/parity/evmbin)                                                                                                                                                     
    Finished dev [unoptimized + debuginfo] target(s) in 14.03s                                                                                                                                                     
     Running /home/sorpaas/repos/parity/target/debug/deps/parity_evm-18f37a82dbda605a

running 7 tests
test display::std_json::tests::should_trace_create_correctly ... ok
test info::tests::should_call_account_from_spec ... ok
test tests::should_parse_all_the_options ... ok
test tests::should_parse_state_test_command ... ok
test display::std_json::tests::should_trace_failure ... ok
test display::json::tests::should_trace_create_correctly ... ok
test display::json::tests::should_trace_failure ... ok

test result: ok. 7 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
```

Reply:
Sorry, I should have push another message instead of the edit of the comment, all is fine :+1: